src/components/app.jsx

```diff
@@ -1,20 +1,37 @@
 import React, { Component } from 'react';
 import './app.css';
 import Status from './status/status';
-import NewsList from './news/news-list/news-list';
+import FaqCreate from './faq/faq-create/faq-create';
+import FaqList from './faq/faq-list/faq-list';
 
 class App extends Component {
-  render() {
-    return (
-      <div className="App">
-        <header className="App-header">
-          <h1 className="App-title">Welcome to React</h1>
-        </header>
-        <Status />
-        <NewsList />
-      </div>
-    );
-  }
+	api(url, method = 'GET', body = {}){
+		const handledStatuses = [200, 201, 204, 400, 401, 404];
+
+		return fetch(url, {
+			method: method,
+			headers: {'Content-Type': 'application/json'},
+			body: body
+		}).then(response => {
+			if (!handledStatuses.includes(response.status)) {
+				throw new Error("Bad request");
```

Comment:
You throws error but don't catch it.
Remove either remove throw or handle this error